pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/KitUploadRoute.java

```diff
@@ -393,10 +396,17 @@ private void addKitRequest(Connection conn, String kitTypeName, KitRequestSettin
                     errorMessage += "collaboratorSampleId was too long ";
                 }
             }
-            KitRequestShipping.writeRequest(ddpInstance.getDdpInstanceId(), shippingId, kitTypeId, kit.getParticipantId().trim(),
-                    collaboratorParticipantId, collaboratorSampleId, userId, addressId, errorMessage, kit.getExternalOrderNumber(), false,
-                    uploadReason, ddpInstance);
-            kit.setDdpLabel(shippingId);
+            if(ddpInstance.getParticipantIndexES() == null){
```

Comment:
I am not sold on that. Can we maybe change it to something more readable?
like adding a new string for the 1 parameter that is changed and setting that per an if ?

Reply:
Got it! I will do that 👍 
\